lib/db.js

```diff
@@ -29,22 +29,7 @@ module.exports = function (
   }
 
   DB.connect = function (options) {
-    var db = new DB(options)
-
-    return db.pool.get('/')
-    .then(
-      function (api) {
-        // TODO: transition to api version
-        // patchLevel is mysql specific
-        if (
-          api.patchLevel < options.patchLevel ||
```

Comment:
So, the returned `api` JSON didn't included a `patchLevel`, and our options don't have a `patchLevel`, so this becomes `undefined < undefined`, which is always false. So this never would throw an error, and currently could only give us a false sense of security. Killed.